chat/src/components/ScoreCard.tsx

```diff
@@ -255,6 +262,7 @@ const ScoreCard = (props: ScoreCardProps) => {
           "text-ellipsis max-w-[100%] break-words space-y-5 leading-normal":
             true,
         }}
+        style={{"-webkit-line-clamp": linesBeforeShowMore}}
```

Comment:
We are using tailwind in this codebase, so please don't use the style param. Rather use the tailwind eqiviland and add it to the styles. i.e. https://tailwindcss.com/docs/line-clamp